example/package.json

```diff
@@ -1,3 +1,7 @@
 {
-  "name": "plop-example"
+  "name": "plop-example",
+  "devDependencies": {
+    "inquirer-directory": "^2.2.0",
+    "plop-pack-fancy-comments": "^0.2.1"
```

Comment:
Not technically breaking, but could break workflows if the user didn't resolve these modules themselves. Making users install their own deps must be done to bundle in any method, but I wanted to flag this.

Reply:
While not a _huge_ breaking change, I'd generally consider these breaking changes.

That said, we need to update `node_plop` to merge in a breaking PR soon anyway:

https://github.com/plopjs/node-plop/pull/192

So we can just include both of these around the same time

Reply:
Understood. It's best to make this breaking then since it wasn't documented thoroughly